packages/x-date-pickers/src/AdapterMomentHijri/AdapterMomentHijri.ts

```diff
@@ -213,27 +213,6 @@ export class AdapterMomentHijri extends AdapterMoment implements MuiPickersAdapt
     return value.clone().iDate(date);
   };
 
-  public getWeekArray = (value: Moment) => {
```

Comment:
The only diff to the two derived moment adapters on this method was the usage of `iMonth` / `jMonth` instead of `month`.
Using `startOfMonth` solves the issue and we can now totally remove this method override.